build/builder.sh

```diff
@@ -6,7 +6,7 @@ image="cockroachdb/builder"
 
 function init() {
   docker build --tag="${image}" - <<EOF
-FROM golang:1.4.2
+FROM golang:1.5
```

Comment:
I'm not sure. Something clearly wasn't working right. It's been a while since I added that TODO.